### DISCUSSION TITLE
About Arabic Translation

### DISCUSSION THREAD

**Discussion by ots25** [borgbase/vorta#1979] [Translations]
I'm the translator of Vorta to Arabic language. I believe that the translation is acceptable and ready for use, after trying it on the latest version of the Arch Linux, but what I didn't understand is that I find untranslated sentences and phrases in the application even though I completed the translation a while ago. What did I miss here?
Also, why translation didn't push to GitHub even though it was ready?  Is there a step required for this?

**Comment by real-yfprojects** [comment]
> What did I miss here?
> Also, why translation didn't push to GitHub even though it was ready? Is there a step required for this?

Translations are manually synced upon a new release. That's why the application won't reflect the newest translations and transifex won't contain all the latest strings that can be found in the application.

**Comment by ots25** [comment]
> and transifex won't contain all the latest strings that can be found in the application.

Where can I find the complete translation source? 
Thank you for your reply
